pkg/loki/loki.go

```diff
@@ -358,6 +359,7 @@ type Loki struct {
 	tenantConfigs             *runtime.TenantConfigs
 	TenantLimits              validation.TenantLimits
 	distributor               *distributor.Distributor
+	ingestLimits              *limits.IngestLimits
```

Comment:
Oh I see, it's because it's registered as a module we have that weird initialization issue?